app/celery/tasks.py

```diff
@@ -31,6 +31,10 @@
 
 @notify_celery.task(name="process-job")
 def process_job(job_id, sender_id=None):
+    """
+    We update job status, get the csv data from s3 and begin processing
+    csv rows.
+    """
```

Comment:
PEP-257 would have this be single-line, which does mean the triple quotes are also on the same line.

`"""Update job status, get csv data from s3, and begin processing csv rows."""`